src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/Licence.kt

```diff
@@ -123,5 +120,50 @@ data class Licence(
     joinColumns = [JoinColumn(name = "licence_id")],
     inverseJoinColumns = [JoinColumn(name = "community_offender_manager_id")]
   )
-  val mailingList: MutableSet<CommunityOffenderManager> = mutableSetOf()
-)
+  val mailingList: MutableSet<CommunityOffenderManager> = mutableSetOf(),
+
+  var variationOfId: Long? = null,
+) {
+  fun createVariation(): Licence {
```

Comment:
Need this function because we only want select fields to be copied over to the variation. For example, `approvedByName` field should not be copied to the new licence